### DISCUSSION TITLE
GoCryptFS FSCK

### DISCUSSION THREAD

**Discussion by ADMECA** [rfjakob/gocryptfs#863] [Q&A]
Hi, i am currently doing a GoCryptFS -fsck on a folder of about 6TB. What do the following messages mean :

"InoMap: opening spillMap for {{45 0} 11466048459870967128}"

and

"unsupported block size of 1048576 bytes, disabling SEEK_DATA & SEEK_HOLE"

Many thanks for your help.

**Comment by rfjakob** [comment]
Both are just info messages.

The first means that the gocryptfs filesystem contains files belonging to different
backing filesystems.

The 2nd means that you have an uncommon block size of the backing
filesystem (you have 1mib, common is 4kib), and file hole reporting has
been disabled. This is just performance feature.

What backing filesystem(s) is this?

**Reply by ADMECA** [reply]
Hi, thanks for the informations.

The storage is on a NAS server in Debian 12, the disks are in software RAID 1, and formatted in EXT4.
This storage is accessible via the Debian CIFS/SAMBA share
